ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -413,6 +415,11 @@ interface TestController {
      * Retrieves a `window` object that corresponds to the currently open window.
      */
     getCurrentWindow(): WindowDescriptorPromise;
+    
+    /**
+     * Retrieves a `Chrome DevTools Protocol` object that corresponds to the currently open window in native automation.
+     */
+    getCurrentCDPSession(): CDPSessionPromise;
```

Comment:
@titerman, please check the description.

Reply:
One minor edit 👍